templates/cloud/plans-and-pricing.html

```diff
@@ -21,6 +21,30 @@
     </div>
 </section>
 
+<section class="row strip-light row-ubuntu-advantage-promo">
+    <div class="strip-inner-wrapper">
+        <div class="eight-col">
+            <h2>Landscape: Server management</h2>
+            <p>
+                Landscape allows you to manage thousands of Ubuntu machines as
+                easily as one, making the administration of Ubuntu desktops,
+                servers and cloud instances more cost-effective. It&rsquo;s
+                easy to set up, easy to use and it requires no special
+                hardware. It features:
+            </p>
+            <ul class="list-ticks--compact">
+                <li>Management at scale</li>
+                <li>Deploy or rollback security updates</li>
+                <li>Live patching</li>
+                <li>Compliance reports</li>
+                <li>Role-based access</li>
+                <li>Informative monitoring</li>
+            </ul>
+            <p><a href="https://landscape.canonical.com/">Learn more about Landscape&nbsp;&rsaquo;</a></p>
+        </div>
+    </div>
+</section>
+
```

Comment:
This seems to be the same copy as the new section in https://github.com/canonical-websites/www.ubuntu.com/pull/1488 could it be a shared include?